src/output-json-alert.c

```diff
@@ -665,6 +666,56 @@ static void AlertAddFrame(const Packet *p, JsonBuilder *jb, const int64_t frame_
     }
 }
 
+/**
+ * \brief    Build verdict object
+ *
+ * \param p  Pointer to Packet current being logged
+ *
+ */
+void GetVerdictJson(JsonBuilder *jb, const Packet *p)
+{
+    jb_open_object(jb, "verdict");
+
+    /* add verdict info */
+    if (PacketCheckAction(p, ACTION_REJECT_ANY)) {
+        // check rule to define type of reject packet sent
+        if (EngineModeIsIPS()) {
+            JB_SET_STRING(jb, "action", "drop");
+        } else {
+            JB_SET_STRING(jb, "action", "alert");
+        }
+        if (PacketCheckAction(p, ACTION_REJECT)) {
+            JB_SET_STRING(jb, "reject_target", "source");
```

Comment:
wonder if it would be clearer to use `to_client` / `to_server` here

Reply:
Isn't it better to keep closer to what the reject keywords have?

Reply:
I think being more explicit about the direction is best, using language we already have. Since we're already thinking about to_server & to_client for most output, I think it makes sense to use it here too